tests/brain/test_attr.py

```diff
@@ -153,36 +153,12 @@ class Eggs:
         @frozen
         class Legs:
             d = attrs.field(default=attrs.Factory(dict))
-
-        m = Legs(d=1)
-        m.d['answer'] = 42
-
-        @define
-        class FooBar:
-            d = attrs.field(default=attrs.Factory(dict))
-
-        n = FooBar(d=1)
-        n.d['answer'] = 42
-
-        @mutable
-        class BarFoo:
-            d = attrs.field(default=attrs.Factory(dict))
-
-        o = BarFoo(d=1)
-        o.d['answer'] = 42
-
-        @my_mutable
-        class FooFoo:
-            d = attrs.field(default=attrs.Factory(dict))
-
-        p = FooFoo(d=1)
-        p.d['answer'] = 42
         """
```

Comment:
Duplicate of line 154 ?

Reply:
No, it was tests newly failing. I'm hoping the new situation is fine.